packages/key-management/src/util/createAsyncKeyAgent.ts

```diff
@@ -1,7 +1,7 @@
 import { AsyncKeyAgent, KeyAgent } from '../';
 import { BehaviorSubject } from 'rxjs';
 
-export const createAsyncKeyAgent = (keyAgent: KeyAgent): AsyncKeyAgent => {
+export const createAsyncKeyAgent = (keyAgent: KeyAgent, shutdownSpy?: () => void): AsyncKeyAgent => {
```

Comment:
```suggestion
export const createAsyncKeyAgent = (keyAgent: KeyAgent, onShutdown?: () => void): AsyncKeyAgent => {
```
Here's a way to ensure we're not introducing code explicitly for tests.

Reply:
Done in https://github.com/input-output-hk/cardano-js-sdk/pull/552/commits/91b242127a566e9f052ca1fcdb9229e0c550e861